google_safe_browsing/help.md

```diff
@@ -4,14 +4,11 @@ Submit URLs to the [Google Safe Browsing](https://safebrowsing.google.com/) Serv
 
 # Key Features
 
-* Feature 1
-* Feature 2
-* Feature 3
+* N/A plugin is obsolete
```

Comment:
We can still list the past Key Features to make it consistent

* URL enrichment